src/ripple/app/rdb/RelationalDBInterface_shards.h

```diff
@@ -35,6 +36,60 @@ struct DatabasePair
 
 /* Shard DB */
 
+/**
+ * @brief makeMetaDBs Opens ledger and transaction 'meta' databases which
+ *        map ledger hashes and transaction IDs to the index of the shard
+ *        that holds the ledger or transaction.
+ * @param config Config object.
+ * @param setup Path to database and opening parameters.
+ * @param checkpointerSetup Database checkpointer setup.
+ * @return Struct DatabasePair which contains unique pointers to the ledger
+ *         and transaction databases.
+ */
+DatabasePair
+makeMetaDBs(
+    Config const& config,
+    DatabaseCon::Setup const& setup,
+    DatabaseCon::CheckpointerSetup const& checkpointerSetup);
+
+/**
+ * @brief saveLedgerMeta Stores (transaction ID -> shard index) and
+ *        (ledger hash -> shard index) mappings in the meta databases.
+ * @param ledger The ledger.
+ * @param app Application object.
+ * @param lgrSesh Session to ledger meta database.
+ * @param txnSesh Session to transaction meta database.
+ * @param shardIndex The index of the shard that contains this ledger.
+ * @return True on success.
+ */
+bool
+saveLedgerMeta(
+    std::shared_ptr<Ledger const> const& ledger,
+    Application& app,
+    soci::session& lgrSesh,
+    soci::session& txnSesh,
+    std::uint32_t const shardIndex);
```

Comment:
Nit: Not necessary/customary to declare this param const.

Reply:
Done